app.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+function StoreSalesGenerator(name, minCust, maxCust, avgSale) {
+  this.storeName = name;
+  this.minCust = minCust;
+  this.maxCust = maxCust;
+  this.avgSale = avgSale;
+  this.numCookies = [];
+  this.totalSales = 0;
+  this.hour = 6;
+}
+
+var newText = document.createTextNode('text');
+var tableRow, tableHeader, tableData;
+var position = document.getElementById('sales-estimates');
+
+var storeHour = function(){
+  tableRow = document.createElement ('tr');
+  position.appendChild(tableRow);
+  tableData = document.createElement ('td');
+  tableRow.appendChild(tableData);
+  tableData = document.createElement ('td');
+  for (var hour = 6; hour < 20; hour++ ) {
+    tableHeader = document.createElement ('th');
+    if (hour < 12 ) {
+      newText = document.createTextNode(hour + ' am ');
+      tableHeader.appendChild(newText);
+      tableRow.appendChild(tableHeader);
+    } else if (hour == 12) {
+      newText = document.createTextNode('12pm ');
+      tableHeader.appendChild(newText);
+      tableRow.appendChild(tableHeader);
+    } else {
+      newText = document.createTextNode((hour-12) + 'pm ');
+      tableHeader.appendChild(newText);
+      tableRow.appendChild(tableHeader);
+    }
```

Comment:
Consider moving this if/else if/else block to a helper function that takes an hour as an argument.  This will allow you to perform a single function call inside the storeHour function in place of the entire if/else block .